benchmarks/benchmark/tools/mlperf/README.md

```diff
@@ -0,0 +1 @@
+In the future, this directory will contain MLPerf related benchmarks.
```

Comment:
Is it possible to just remove the directory and add it when MLPerf releated benchmarks are available? An empty dir with just a README is misleading.

Reply:
Done